app/controllers/games_controller.rb

```diff
@@ -1,8 +1,8 @@
 class GamesController < ApplicationController
-  before_action :authenticate_user!, only: [:new, :create, :index]
+  before_action :authenticate_user!, only: [:new, :create, :index, :update]
 
   def index
-    @games = Game.all
+    @games = Game.where("jedi_user_id is null or sith_user_id is null") 
```

Comment:
This is great.  For a refactor, you might want to look into `named scopes` and moving the SQL stuff into the model.  Also _SUPER_ nitpick, but usually when doing SQL stuff, `null` is spelled in all caps: `NULL` (but it works both ways).